dts/bindings/i3c/snps,designware-i3c.yaml

```
@@ -0,0 +1,32 @@
+# Copyright (C) 2020 Samsung Electronics Co., Ltd.
+# Copyright (C) 2023 Meta Platforms
+# SPDX-License-Identifier: Apache-2.0
+
+description: >
+    This is a binding for the Synopsys I3C interface
+
+compatible: "snps,designware-i3c"
```

shouldn't we maintain compatiblity with Linux?, ie

```suggestion
compatible: "snps,dw-i3c-master-1.00a"
```

---

I'm not sure if that is necessary, I'm going on constancy here. like how `snps,designware-i2c` is already defined

---

yeah..., but this is not the 1.00a version of the IP this driver is supporting here. This is the 1.01a which does include some features that are within this driver.

---

so then we need to specify IP version in the compatible, we, for example, have 1.00a. Driver should be able to support both, based on the DT compatible.

---

looking again through the databook, it looks like both should be supported anyways here

---

I think this is still valid, we have a rule that says we should align with upstream Linux when possible.

---

but then there is a rule to not introduce 'offensive terms' with the word `master`. :shrug: 
https://docs.zephyrproject.org/latest/contribute/coding_guidelines/index.html#rule-a-2-inclusive-language

I'm still not convinced that linux made the right choice here with the naming of it especially since this driver doesn't only support the "controller" implemention of the IP. I'll poke around on the linux mailing lists to see what response I get and update the PR depending on what I get.